rust/scx_rustland_core/assets/bpf.rs

```diff
@@ -246,7 +246,6 @@ impl<'cb> BpfScheduler<'cb> {
 
         skel.bss_mut().usersched_pid = std::process::id();
         skel.rodata_mut().slice_ns = slice_us * 1000;
-        skel.rodata_mut().switch_partial = partial;
```

Comment:
We should also drop `partial` from `BpfScheduler`, update scx_rustland and scx_rlfifo, and drop the option `--partial` from scx_rustland), but it can be done later in a separate commit (this is just as a reminder).

Reply:
`--partial` still sets `SCX_OPS_SWITCH_PARTIAL` in `sched_ext_ops.flags`, so it should keep working the same.